src/appfl/run_mpi.py

```diff
@@ -243,7 +123,7 @@ def run_server(
         accuracy = 0
         BestAccuracy = 0
         if cfg.validation == True:
-            test_loss, accuracy = validation(server, server_dataloader)
+            test_loss, accuracy = validation(server, copy.deepcopy(server.model), test_dataloader)
```

Comment:
This looks like copying the model every epoch, which may be expensive. Can we avoid this?

Reply:
Yes. We can insert "model.train()" for every iteration.

Reply:
removed.